packages/browser/src/index.ts

```diff
@@ -159,8 +159,13 @@ export default class LogtoClient {
       return accessToken.token;
     }
 
+    // Since the access token has expired, delete it from the map.
+    if (accessToken) {
+      this.accessTokenMap.delete(accessTokenKey);
```

Comment:
per our discussion in the previous PR, this will cause duplicate removal.

Reply:
Tested `expect(accessTokenMap.delete).toBeCalledTimes(1);` above.
Seem that it will not cause duplicate removal.

Maybe the test case above is false.